index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="main">
+        <header>
+            <div class="normal_button">
+                <img src="images/Icon - Menu.png" width="24px" height="24px" alt="Menu-Icon">
+            </div>
+        </header>
+        <section class="headline">
+            <h2 class="greeting">Hey there.<br>I’m Alex.</h2>
```

Comment:
- [ ] This is the main title on your page and the appropriate heading tag for it is `h1`. If you want that google (and other search engines) knows about a specific topic, in this case, you use an `h1` tag. You can check this [article](https://www.reliablesoft.net/h1-tag/) for more information.

Reply:
it was mentioned as h2 in the figma template so I used h2 instead of h1. But I'll change it.